app/Base/types/project.ts

```diff
@@ -1,5 +1 @@
-export type Project = {
```

Comment:
This is not required right? Lets remove this.

Reply:
This is used in one of the check permissions function